components/org.wso2.carbon.identity.api.server.tenant.management/org.wso2.carbon.identity.api.server.tenant.management.v1/src/main/java/org/wso2/carbon/identity/api/server/tenant/management/v1/core/ServerTenantManagementService.java

```diff
@@ -241,6 +274,73 @@ private List<OwnerResponse> createOwnerResponse(User user) {
         return ownerResponseList;
     }
 
+    private Tenant createTenantInfoBean(String tenantUniqueID, String ownerID, OwnerPutModel ownerPutModel)
+            throws TenantMgtException {
+
+        Tenant tenant = TenantManagementServiceHolder.getTenantMgtService().getTenant(tenantUniqueID);
+        if (tenant.getAdminUserId() == null || !tenant.getAdminUserId().equals(ownerID)) {
+            throw handleException(Response.Status.NOT_FOUND, TenantManagementConstants.ErrorMessage.
+                    ERROR_CODE_OWNER_NOT_FOUND, ownerID);
+        }
+        if (StringUtils.isNotBlank(ownerPutModel.getFirstname())) {
+            tenant.setAdminFirstName(ownerPutModel.getFirstname());
+        }
+        if (StringUtils.isNotBlank(ownerPutModel.getLastname())) {
+            tenant.setAdminLastName(ownerPutModel.getLastname());
+        }
+        if (StringUtils.isNotBlank(ownerPutModel.getEmail())) {
```

Comment:
No email validation required ? or is it validated earlier ?

Reply:
Email validation is done inside the multitenancy repo.